client/src/telemetry/telemetry.ts

```diff
@@ -157,4 +164,14 @@ export class TelemetryService {
 		return util.format('%d%d', hrend[0], hrend[1] / 1000000);
 	}
 
+	public async checkCliTelemetry(): Promise<boolean> {
+		return await sfdxCoreExtension.exports.isCLITelemetryAllowed(sfdxCoreExtension.exports.getRootWorkspacePath());
```

Comment:
Should this actually be `sfdxCoreExtension.exports.telemetryService.isCLITelemetryAllowed`? Also could you manually verify with SFDX CLI, and set breakpoints here to check if telemetry is disabled.

cc @rcoringrato-sfdc

Reply:
hey @xyc. thanks for your comment. you are probably correct. i am trying to verify under what namespace these methods are available. i'll be running some tests to confirm.

Reply:
@xyc finished manual testing on my end and it is working as expected. ready to review